templates/base/client/.rtag/App.tsx.hbs

```diff
@@ -79,44 +90,120 @@ function App() {
           <Route path="*" element={<div>Not Found</div>} />
         </Routes>
       </div>
-    </>
+    </div>
+  );
+}
+
+type NavBarProps = {
+  token: string | null;
+};
+function NavBar({ token }: NavBarProps) {
+  const navigate = useNavigate();
+  const user = token ? RtagClient.getUserFromToken(token) : null;
+
+  return (
+    <Disclosure as="nav" className="bg-indigo-500">
```

Comment:
why is this wrapped in a `Disclosure`?

Reply:
I've removed `Disclosure`, it was only needed for other stuff that has since been removed, thanks for catching that.